examples/iam/openidconnectprovider.yaml

```
@@ -1,15 +1,11 @@
 apiVersion: iam.aws.upbound.io/v1beta1
 kind: OpenIDConnectProvider
 metadata:
```

nit: Although we are not currently using the `example-id` annotation for its intended purpose, it would be good to add it for compatibility.

```suggestion
metadata:
  annotations:
    meta.upbound.io/example-id: iam/v1beta1/openidconnectprovider
```